src/views/GovernanceTaskList/TaskListItem.test.tsx

```diff
@@ -47,60 +42,17 @@ describe('The TaskListItem', () => {
     ).toEqual(true);
   });
 
-  it('displays start button', () => {
+  it('displays children', () => {
     const component = shallow(
       <TaskListItem
         heading="Test Heading"
         description="Test Description"
         status="START"
-        link="/"
-      />
-    );
-
-    expect(
-      component
-        .find(UswdsLink)
-        .dive()
-        .text()
-    ).toEqual('Start');
-  });
-
-  it('displays continue button', () => {
-    const component = shallow(
-      <TaskListItem
-        heading="Test Heading"
-        description="Test Description"
-        status="CONTINUE"
-        link="/"
-      />
+      >
+        <div id="test-div">Test</div>
+      </TaskListItem>
     );
 
-    expect(
-      component
-        .find(UswdsLink)
-        .dive()
-        .text()
-    ).toEqual('Continue');
-  });
-
-  describe('submitted request link', () => {
-    const component = shallow(
-      <TaskListItem
-        heading="Test Heading"
-        description="Test Description"
-        status="COMPLETED"
-        link="/system-intake/FOO"
-      />
-    );
-
-    it('displays the right text', () => {
-      expect(component.find(Link).text()).toEqual(
-        'View Submitted Request Form'
-      );
-    });
-
-    it('displays the right link', () => {
-      expect(component.find(Link).prop('to')).toEqual('/system-intake/FOO');
-    });
+    expect(component.find('#test-div').text()).toEqual('Test');
```

Comment:
Nice use of the html `id` prop here! If you don't really care what it looks like, just that it exists, you could also do: `expect(component.find('#test-div').exists()).toBe(true)`